packages/test-commerce-sdk-react/package.json

```diff
@@ -24,15 +24,15 @@
     "@types/react-dom": "^18.2.1",
     "@types/react-router-dom": "^5.3.3",
     "@salesforce/commerce-sdk-react": "1.0.0-dev",
-    "internal-lib-build": "^3.0.0-dev",
-    "@salesforce/pwa-kit-dev": "^3.0.0-dev",
-    "@salesforce/pwa-kit-react-sdk": "^3.0.0-dev",
-    "@salesforce/pwa-kit-runtime": "^3.0.0-dev",
+    "internal-lib-build": "3.0.0-dev",
+    "@salesforce/pwa-kit-dev": "3.0.0-dev",
+    "@salesforce/pwa-kit-react-sdk": "3.0.0-dev",
+    "@salesforce/pwa-kit-runtime": "3.0.0-dev",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-helmet": "^6.1.0",
     "react-router-dom": "^5.3.4",
-    "typescript": "4.8.3"
+    "typescript": ">=4.8.3"
```

Comment:
Using greater than here meaning we accept any version that is greater than 4.8.2 including version 5? We have to think a bit careful about this because there are some breaking changes in 4.9
https://github.com/microsoft/TypeScript/wiki/Breaking-Changes